account-kit/infra/src/middleware/gasManager.ts

```diff
@@ -137,10 +193,8 @@ export function alchemyGasAndPaymasterAndDataMiddleware(
       }
 
       // Fall back to the default 7677 dummyPaymasterAndData middleware.
-      return alchemyGasManagerMiddleware(policyId).dummyPaymasterAndData!(
-        uo,
-        args
-      );
+      return alchemyGasManagerMiddleware(policyId, policyToken)
+        .dummyPaymasterAndData!(uo, args);
```

Comment:
can remove this here now too since you added the `Required`
```suggestion
        .dummyPaymasterAndData(uo, args);
```